ra-tls/ra-tls-server/run.go

```diff
@@ -68,6 +69,11 @@ EXAMPLE:
 			addr = defaultAddress
 		}
 
+		err := syscall.Unlink(addr)
```

Comment:
If the sock file doesn't exist, this check will cause ra-tls-server going down forever.

Reply:
Thank you, you are right!